src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java

```diff
@@ -589,4 +593,133 @@ private static StringBuilder getHearingsToJudgmentsReport(ListingData listingDat
         return sb;
     }
 
+    private static StringBuilder getHearingsByHearingTypeReport(ListingData listingData) {
+        var sb = new StringBuilder();
+        AdhocReportType localReportSummaryHdr = listingData.getLocalReportsSummaryHdr();
+        AdhocReportType localReportSummaryHdr2 = listingData.getLocalReportsSummaryHdr2();
+        if (localReportSummaryHdr != null) {
+            sb.append(getHearingCounts(localReportSummaryHdr));
+        }
+        List<AdhocReportTypeItem> localReportSummary = listingData.getLocalReportsSummary();
+
+        if (!CollectionUtils.isEmpty(localReportSummary)) {
```

Comment:
Minor point but change to isNotEmpty? Same with the others

Reply:
done